tensorflow/lite/micro/micro_graph.cc

```diff
@@ -244,4 +241,14 @@ TfLiteEvalTensor* MicroGraph::GetSubgraphOutput(int subgraph_idx,
   return &subgraph_allocations_[subgraph_idx].tensors[tensor_idx];
 }
 
+int MicroGraph::NumSubgraphOperators(int subgraph_idx) {
```

Comment:
there isn't any reason for this function to be a public member of the MicroGraph class.

Can we make it a free function:
```cc
int NumSubgraphOperators(const Subgraph& subgraph);
```

And have this function be used across micro_graph.cc and micro_allocator.cc?

I'd be ok with putting this and the code from https://github.com/tensorflow/tflite-micro/pull/243 into a flatbuffer_utils.h/flatbuffer_utils.cc module.

Reply:
Is this the path you had in mind:
tensorflow/lite/micro/flatbuffer_utils.*

Reply:
yup

Reply:
I think this function is now unused and can be removed?